test/merge_test_old.js

```diff
@@ -1,269 +0,0 @@
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-/**
- * @fileoverview Unit tests for merge().
- * @author bnkuhn@gmail.com (Brian Kuhn)
- */
-
-test('merge', function() {
-
-  function assertMerge(from, to, expected) {
-    var fromCopy = helper.isArray_(from) ? [] : {};
-    jQuery.extend(true, fromCopy, from);
-    helper.merge_(from, to);
-    deepEqual(from, fromCopy);
-    deepEqual(to, expected);
-  }
-
-  // If <from> property is array and <to> property is "other", <to> property
-  // will be overwritten.
-  assertMerge({a: []}, {}, {a: []});
-  assertMerge({a: []}, {a: 1}, {a: []});
-  assertMerge({a: []}, {b: 2}, {a: [], b: 2});
-  assertMerge({a: []}, {a: 1, b: 2}, {a: [], b: 2});
-  assertMerge({a: [1, 2]}, {}, {a: [1, 2]});
-  assertMerge({a: [1, 2]}, {a: 1}, {a: [1, 2]});
-  assertMerge({a: [1, 2]}, {b: 2}, {a: [1, 2], b: 2});
-  assertMerge({a: [1, 2]}, {a: 1, b: 2}, {a: [1, 2], b: 2});
-  assertMerge({a: [], b: []}, {}, {a: [], b: []});
-  assertMerge({a: [], b: []}, {a: 1, b: 2}, {a: [], b: []});
-  assertMerge({a: [], b: []}, {a: 1, b: 2, c: 3}, {a: [], b: [], c: 3});
-
-  // If <from> property is array and <to> property is "array", <from> property
-  // will be merged into <to> property.
-  assertMerge({a: []}, {a: []}, {a: []});
-  assertMerge({a: []}, {a: [1]}, {a: [1]});
-  assertMerge({a: []}, {a: [undefined, 2]}, {a: [undefined, 2]});
-  assertMerge({a: [1]}, {a: [undefined, 2]}, {a: [1, 2]});
-  assertMerge({a: [1, 2]}, {a: [undefined, 2]}, {a: [1, 2]});
-  assertMerge({a: [1, 2, 3]}, {a: [undefined, 2]}, {a: [1, 2, 3]});
-  assertMerge({a: [1, 3, 3]}, {a: [undefined, 2]}, {a: [1, 3, 3]});
-  assertMerge({a: [1, 2, 3]}, {a: []}, {a: [1, 2, 3]});
-
-  // If <from> property is plain object and <to> property is "other",
-  // <to> property will be overwritten.
-  assertMerge({a: {}}, {}, {a: {}});
-  assertMerge({a: {}}, {a: 1}, {a: {}});
-  assertMerge({a: {}}, {b: 2}, {a: {}, b: 2});
-  assertMerge({a: {}}, {a: 1, b: 2}, {a: {}, b: 2});
-  assertMerge({a: {x: 1, y: 2}}, {}, {a: {x: 1, y: 2}});
-  assertMerge({a: {x: 1, y: 2}}, {a: 1}, {a: {x: 1, y: 2}});
-  assertMerge({a: {x: 1, y: 2}}, {b: 2}, {a: {x: 1, y: 2}, b: 2});
-  assertMerge({a: {x: 1, y: 2}}, {a: 1, b: 2}, {a: {x: 1, y: 2}, b: 2});
-  assertMerge({a: {}, b: {}}, {}, {a: {}, b: {}});
-  assertMerge({a: {}, b: {}}, {a: 1, b: 2}, {a: {}, b: {}});
-  assertMerge({a: {}, b: {}}, {a: 1, b: 2, c: 3}, {a: {}, b: {}, c: 3});
-
-  // If <from> property is plain object and <to> property is plain object,
```

Comment:
I did not see tests equivalent to this section in the new test suite.

Reply:
Thanks for pointing it out, I can't believe I missed that... With the last push, all the tests should have been migrated.